tests/e2e/logging/test_openshift-logging.py

```diff
@@ -223,8 +225,8 @@ def test_respin_osd_pods_to_verify_logging(self, create_pvc_and_deploymentconfig
         project1_filecount = elasticsearch_pod_obj.exec_cmd_on_pod(
             command=f'es_util --query=project.{project1}.*/_count'
         )
-        assert project1_filecount['_shards']['successful'] == 0, f"No files found in project {project1}"
-        logger.info(f'The files in the project 1 {project1_filecount}')
+        assert project1_filecount['_shards']['successful'] != 0, f"No files found in project {project1}"
```

Comment:
What is the expected value of project1_filecount['_shards']['successful'] ? If there is a specific value, check can be done for that particular value. 

Value from last passed job - {"count":1,"_shards":{"total":3,"successful":3,"skipped":0,"failed":0}}